### PR TITLE
Show onlyMembers over minScore message and make block more visible

### PR DESCRIPTION
**Pull request by samuveth**
Fixes https://github.com/snapshot-labs/snapshot/issues/1581

Changes proposed in this pull request:
- Use link color for text and border instead of text color
- Show onlyMembers over minScore message
- Changed the Plugins and Validations modal layout to be the same as the Strategies modal
- Hide validation and threshold input if onlyMembers

### CODE REVIEW

File: src/views/SpaceCreate.vue [hunk 1]
```diff
@@ -247,17 +247,27 @@ watchEffect(() => {
...
-                  formatCompactNumber(space.filters.minScore),
-                  space.symbol
-                ])
-              : $t('create.validationWarning.basic.member')
-          }}
+          <span v-if="space?.filters.onlyMembers">
+            {{ $t('create.validationWarning.basic.member') }}
+          </span>
+          <span
+            v-else-if="
```
Comment (midgerate) [inline src/views/SpaceCreate.vue:260]: What happens when authors also have to meet the threshold?

Comment (samuveth) [reply src/views/SpaceCreate.vue:260]: They don't

**Comment by bonustrack** [comment]
Selecting a validation doesn't work for me

**Comment by bonustrack** [comment]
I think we can disable the fields "Validation" and "Proposal threshold" if the option "Only member can post.." is checked

**Comment by midgerate** [comment]
@bonustrack agree

**Comment by samuveth** [comment]
Changed the Plugins and Validations modal layout to be the same as the Strategies modal.

Before:
image

After: 
image

Also planning on fixing the Textarea scroll bar, but that would add to much to this PR.

**Review by midgerate — APPROVED** [review]

**Pull request merged**